reference/sites.v3.yml

```diff
@@ -653,31 +694,47 @@ components:
       x-internal: false
     _site:
       type: object
+      title: _site
+      x-internal: false
       properties:
-        id:
-          type: integer
-        url:
-          type: string
-          description: The Fully Qualified URL (including host and scheme) where this site is hosted. All URLs generated for this site will be appended to this.
-          example: 'http://kittens.mybigcommerce.com/'
         channel_id:
           type: integer
           description: 'The channel this site is attached to. Each site belongs to a single channel, and each channel can have either zero or one sites.'
         created_at:
           type: string
           description: When was this site created? RFC 3339
-          example: '2018-01-04T04:15:50.000Z'
+          format: date-time
```

Comment:
is this `format: date-time` a standard thing? if so, add to the other RFC-3339 fields.